src/yb/rocksdb/db/db_impl.cc

```diff
@@ -3431,6 +3435,9 @@ void DBImpl::BackgroundCallFlush(ColumnFamilyData* cfd) {
   bool made_progress = false;
   JobContext job_context(next_job_id_.fetch_add(1), true);
 
+  SET_WAIT_STATE_METADATA(yb::util::AUHMetadata{.request_id = std::to_string(next_job_id_.load())});
```

Comment:
Instead of replacing the whole AUHMetadata, we could just  update the request_id accordingly:

`yb::util::WaitStateInfo::CurrentWaitState().metadata().request_id = ...`